CHANGELOG_PENDING.md

```diff
@@ -13,6 +13,8 @@ Deprecations
 
 SDK Features
 ---
+* `service/dynamodb/dynamodbattribute`: New Encoder and Decoder Behavior for Empty Collections
```

Comment:
Nit: Add PR reference

Reply:
Also would be great to reference the V1 associated change